packages/beacon-node/src/network/reqresp/inboundRateLimiter.ts

```diff
@@ -44,13 +43,13 @@ export class InboundRateLimiter {
   private readonly logger: ILogger;
   private readonly reportPeer: RateLimiterModules["reportPeer"];
   private readonly metrics: IMetrics | null;
-  private requestCountTrackersByPeer: MapDef<string, RateTracker>;
+  private requestCountByPeerTracker: RateLimiterGRCA<string>;
   /**
    * This rate tracker is specific to lodestar, we don't want to serve too many blocks for peers at the
    * same time, even through we limit block count per peer as in blockCountTrackersByPeer
    */
-  private blockCountTotalTracker: RateTracker;
-  private blockCountTrackersByPeer: MapDef<string, RateTracker>;
+  private blockCountTotalTracker: RateLimiterGRCA<null>;
+  private blockCountByPeerTracker: RateLimiterGRCA<string>;
```

Comment:
Change this to be generic per method. Add a quota per overall method calls, and per method items. Move the rate limiter to the reqResp package and make it generic to any route. 

```
interface RateLimiterOpts {
  totalRequestsQuota?: RateLimiterQuota
  methodsQuota?: Record<string, {
    requestsQuota: RateLimiterQuota;
    itemsQuota?: RateLimiterQuota;
  }>
}
```